tables/output_tables.Rmd

```diff
@@ -146,13 +146,13 @@ soi_df <- tmb_hist %>%
   filter(experimental_strategy == "WGS") %>%
   # manually add therapy field and update cancer predispositions not included in histologies file per [ticket](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1318)
   mutate(`Therapy post-biopsy` = case_when(
-      Kids_First_Biospecimen_ID == "BS_VW4XN9Y7" ~ "Temozolomide, CCNU, Radiation",
-      Kids_First_Biospecimen_ID == "BS_20TBZG09" ~ "Temozolomide, Irinotecan, Bevacizumab, Radiation",
+      Kids_First_Biospecimen_ID == "BS_VW4XN9Y7" ~ "Radiation, Temozolomide, CCNU",
+      Kids_First_Participant_ID == "PT_3CHB9PK5" ~ "Radiation, Temozolomide, Irinotecan, Bevacizumab",
```

Comment:
I'm not sure I understand why this ID was changed from biospecimen to patient?

Reply:
There's no impact on the table outside of the intended impact, which is to include previous treatment for the progressive BS ID (`BS_8AY2GM4G`) in addition to the Initial CNS Tumor (`BS_20TBZG09`).